From 9abbecc0c1965ed13385a17e143400da48f2216a Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 18 Oct 2017 17:29:20 +0800
Subject: [PATCH 2665/5242] MLK-16603: dtb: Disable imx8qm native hdmi/dp
 driver in it6263 dtb

commit  26e28f3d48a2d08428877bd2c6d31c6cdc336bdc from
https://source.codeaurora.org/external/imx/linux-imx.git

Only 25% iMX8QM SOC chip can support DP/HDMI function now.
We may not find enough hdmi/dp work SOC chip for everyone.
So lvds-hdmi display is the prime display for iMX8QM ARM2 board.
Disable imx8qm native hdmi/dp driver in it6263 dtb
make sure lvds-hdmi display is working with
hdmi/dp function failed chip.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit d4235299d583126edb5996e008be5366590252ee)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../freescale/fsl-imx8qm-lpddr4-arm2-it6263.dts    |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-it6263.dts
index c88d4fc..d444d17 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-it6263.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-it6263.dts
@@ -101,3 +101,7 @@
 		};
 	};
 };
+
+&hdmi {
+	status = "disabled";
+};
-- 
1.7.9.5

